tests/jobs/rft/data/norne/expected_result/RFT_B-4AH_0_active

```diff
@@ -0,0 +1,29 @@
+1
+1
+1
+1
+1
+1
```

Comment:
Are these files really necessary to check in? Can't they be mocked?

Reply:
Let's add them in the test, fixing

Reply:
added